chain-impl-mockchain/src/testing/e2e/fees.rs

```diff
@@ -239,3 +244,93 @@ fn verify_total_funds_after_transaction_with_fee(fee: u64) {
         .pots()
         .has_fee_equals_to(&Value(fee));
 }
+
+#[quickcheck]
+pub fn vote_cast_fees(constant_fee: u64, coefficient: u64, certificate: u64) {
```

Comment:
I'm pretty sure that `LinearFee` implements Arbitrary, so you can write directly:

`pub fn vote_cast_fees(linear_fee: LinearFee) {`